articles.tex

```diff
@@ -303,6 +303,8 @@
 	\item To act as a Judicial Board as defined in \ref{Judicial}.
 	\item To review major projects, as defined in \ref{Expectations of House Members},  presented to them by the Evaluations director.
 	\item To make the final vote regarding conditionals and appeals as defined in \ref{Membership Evaluation}.
+	\item To proactively maintain a safe and welcoming enviornment to House members.
+	\item To respect the privacy of House members who confide in confidence barring circumstances that allude endangerment of ones self or others.
```

Comment:
I think this is much easier stated "To respect the privacy of House members who confide in them" I don't think it's necessary to provide a specific opposite case as it can be considered implied (Not to mention any E-Board members who may be mandatory reporters under Title IX )